.rubocop.yml

```diff
@@ -3,4 +3,4 @@ inherit_gem:
 
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.6
```

Comment:
I think that target rubocop version should be the minimum supported version of MRI

Reply:
Sure, otherwise it's going to suggest features that are not supported yet.

Reply:
I agree